prisma/schema.prisma

```diff
@@ -60,9 +59,9 @@ model VerificationToken {
 }
 
 model ScopesOnIssues {
-  issue   Issue  @relation(fields: [issueId], references: [id], onUpdate: NoAction)
+  issue   Issue  @relation(fields: [issueId], references: [id], onUpdate:Restrict)
```

Comment:
I don't find any information to explain this change, why it's needed ?

Reply:
when i started changing the configuration , vsCode said it was an error and i should change it . 
now NoAction exists , it was weird .